website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -86,53 +86,85 @@ HCP Terraform prioritizes and overwrites conflicting variables according to the
 
-### 4. Command line argument variables
+### 6. Command line argument variables
 
 When using a CLI workflow, variables applied to a run with either `-var` or `-var-file` overwrite workspace-specific and variable set variables that have the same key.
 
-### 5. Local environment variables prefixed with `TF_VAR_`
+### 7. Local environment variables prefixed with `TF_VAR_`
 
 When using a CLI workflow, local environment variables prefixed with `TF_VAR_` (e.g., `TF_VAR_replicas`) overwrite workspace-specific, variable set, and `.auto.tfvars` file variables that have the same key.
 
-### 6. Workspace-specific variables
+### 8. Workspace-specific variables
 
 Workspace-specific variables always overwrite variables from variable sets that have the same key. Refer to [overwrite variables from variable sets](/terraform/cloud-docs/workspaces/variables/managing-variables#overwrite-variable-sets) for details.
 
-### 7. Workspace-scoped variable sets
+### 9. Workspace-scoped, project-owned variable sets
 
 Variables in workspace-scoped variable sets are only applied to a subset of workspaces in an organization.
 
 When workspace-scoped variable sets have conflicting variables, HCP Terraform compares the variable set names and uses values from the variable set with lexical precedence. Terraform and HCP Terraform operate on UTF-8 strings, and HCP Terraform sorts variable set names based on the lexical order of Unicode code points.
 
 For example, if you apply `A_Variable_Set` and `B_Variable_Set` to the same workspace, HCP Terraform will use any conflicting variables from `A_Variable_Set`. This is the case regardless of which variable set has been edited most recently. HCP Terraform only considers the lexical ordering of variable set names when determining precedence.
 
-### 8. Project-scoped variable sets
+Variables sets owned by projects take precedence over those owned by organizations.
+
+### 10. Workspace-scoped, organization-owned variable sets
+
+Variables in workspace-scoped variable sets are only applied to a subset of workspaces in an organization.
+
+When workspace-scoped variable sets have conflicting variables, HCP Terraform compares the variable set names and uses values from the variable set with lexical precedence. Terraform and HCP Terraform operate on UTF-8 strings, and HCP Terraform sorts variable set names based on the lexical order of Unicode code points.
+
+For example, if you apply `A_Variable_Set` and `B_Variable_Set` to the same workspace, HCP Terraform will use any conflicting variables from `A_Variable_Set`. This is the case regardless of which variable set has been edited most recently. HCP Terraform only considers the lexical ordering of variable set names when determining precedence.
+
+### 11. Project-scoped, project-owned variable sets
+
+Workspace-specific variables and workspace-scoped variable sets always take precedence over project-scoped variable sets that are applied to workspaces within a project.
```

Comment:
I don't think we need this sentence because the section ordering makes it obvious that workspace-scoped variables take precedence? Curious how you feel though!

Reply:
That makes sense to me too! This line in particular is kind of confusing too now that I'm reading it because the section is about project scoped variables but then we start this sentence off talking about workspace scoped variables.

Reply:
```suggestion
```